src/vma/dev/qp_mgr.cpp

```diff
@@ -774,6 +775,25 @@ int qp_mgr_eth::prepare_ibv_qp(vma_ibv_qp_init_attr& qp_init_attr)
 	}
 	BULLSEYE_EXCLUDE_BLOCK_END
 
+	int attr_mask = IBV_QP_CAP;
```

Comment:
The entire following block (including setting and logging `m_max_inline_data`) was originally done in the qp_mgr constructor for both Eth and IB. Please do this for IB as well.

#### Introduced in:
MP_RQ: add cq_mgr_mp and qp_mgr_mp